ggshield/verticals/sca/output/schemas.py

```diff
@@ -0,0 +1,12 @@
+from ggshield.verticals.sca.sca_scan_models import (
+    SCAScanAllOutputSchema,
+    SCAScanDiffOutputSchema,
+)
+
+
+class JSONSCAScanAllOutputSchema(SCAScanAllOutputSchema):
```

Comment:
nit: 
```suggestion
class SCAJSONScanAllOutputSchema(SCAScanAllOutputSchema):
```

to follow the naming for IaC